albumentations/augmentations/transforms.py

```diff
@@ -953,6 +939,8 @@ def get_transform_init_args_names(self):
 
 class RandomSizedBBoxSafeCrop(DualTransform):
     """Crop a random part of the input and rescale it to some size without loss of bboxes.
+    Keypoints are processed but does not affect selection of crop region. So if you need to crop preserving
+    keypoints you have to pass fictive bbox enclosing keypoints.
```

Comment:
Thank you, great work! 
But I don't understand this clarification. Can you clarify this with an example?